lib/jwe/encrypt.js

```diff
@@ -369,7 +369,7 @@ function JWEEncrypter(cfg, fields, recipients) {
           return jwe;
         } else if (props.zip === "DEF") {
           return new Promise(function(resolve, reject) {
-            zlib.deflateRaw(Buffer.from(pdata, "binary"), function(err, data) {
+            pako.deflateRaw(Buffer.from(pdata, "binary"), function(err, data) {
```

Comment:
Tests are failing here.  From a glance, it appears `pako.deflateRaw` is synchronous, not callback-style asynchronous.